GoMain/src/main/main.go

```diff
@@ -164,9 +161,5 @@ func orchestrationInit() error {
 
 	log.Println(logPrefix, "orchestration init done")
 
-	// TODO remove this line
```

Comment:
@suresh-lc LGTM. What do you think of deprecating the hard-coded `Sleep` operation?